easybuild/easyblocks/v/vmd.py

```diff
@@ -45,6 +46,15 @@
 class EB_VMD(ConfigureMake):
     """Easyblock for building and installing VMD"""
 
+    def extract_step(self):
+        """Custom extract step for VMD."""
+        super(EB_VMD, self).extract_step()
+
+        vmddir = os.path.join(self.src[0]['finalpath'], '%s-%s' % (self.name.lower(), self.version))
+        if LooseVersion(self.version) >= LooseVersion("1.9.3"):
+            change_dir(os.path.join(vmddir, 'lib', 'surf'))
+            extract_file('surf.tar.Z', os.getcwd())
```

Comment:
@akesandgren Any reference to what this is exactly? Will it always be there?

Reply:
There is something in the docco about it but i can't remember, it's something users sometimes need.
As of that version they moved it into VMD so should always be there. Unless they change their mind...

Reply:
page 4 of the install guide says that since surf is free it is provided as is in the distro of VMD.
As for what it really is.... no idea.

Reply:
Can we include a URL in a comment here that provides some context?

If it's not there at some point, we'll notice quickly enough.